styles/_top-nav.scss

```diff
@@ -418,16 +424,21 @@
           }
 
           // "Docs" dropdown placement
-          .col-for-article-types {
+          .col-for-article-types,
+          .col-for-カテゴリ
+           {
             grid-column: 1;
             grid-row: 2;
           }
-          .col-for-online-tools {
-            grid-column: 1;
-            grid-row: 3;
-            padding-bottom: 32px;
-          }
-          .col-for-get-started {
+          // Not used
```

Comment:
Good point that the `.col-for-online-tools` style is no longer used. We should remove it entirely rather than comment it out (here and above)